web/styles/modal.css

```diff
@@ -445,9 +445,13 @@
 #add-poll-modal {
     /* this height allows 3-4 option rows
     to fit in without need for scrolling */
-    height: 450px;
+    height: 32.1428em; /* 450px / 14px em */
     overflow: hidden;
 
+    .modal__title {
+        font-size: 1.375em; /* Replace default rem */
```

Comment:
This probably should be updated for all modals, but I haven't tested all modals with this change yet.

Reply:
https://chat.zulip.org/#narrow/channel/431-redesign-project/topic/broader.20range.20of.20font.20sizes/near/2030155

Reply:
Removed in #32957

Reply:
Amazing thanks!